pkg/apis/sources/v1alpha1/register.go

```diff
@@ -31,29 +31,31 @@ import (
 )
 
 var (
-	// SchemeGroupVersion is group version used to register these objects
+	// SchemeGroupVersion is group version used to register these objects.
 	SchemeGroupVersion = schema.GroupVersion{Group: sources.GroupName, Version: "v1alpha1"}
```

Comment:
this sources refers to "knative.dev/eventing/pkg/apis/sources". See the import statement. 
You should be importing ours which has our group name to be sources.google.com

Reply:
check that we are not importing anywhere "knative.dev/eventing/*"

Reply:
Are you refering to this ? 
https://github.com/google/knative-gcp/blob/49ed3cf0f8e2e30a740a2254dfffd6ca4f0364f2/pkg/apis/events/v1beta1/register.go#L24-L28

Reply:
"github.com/google/knative-gcp/pkg/apis/events"

Reply:
make sure it reads the crorect one